app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -147,6 +151,26 @@
         html = dateSelector("end", "edit-set.record.edit.end-date"),
         role = Some("group")
     ))
+@govukSelect(Select(
+    id = "legalStatus",
+    name = "legalStatus",
+    items = legalStatusData.map {status =>
```

Comment:
This is where I'd put the blank option, with a message lookup.

Reply:
Won't blank option be part of the list?